samples/charts/doughnut-chart/explosion/src/DoughnutChartExplosion.ts

```diff
@@ -8,70 +10,42 @@ import { ModuleManager } from 'igniteui-webcomponents-core';
 
 ModuleManager.register(
     IgcDoughnutChartModule,
-    IgcRingSeriesModule
+    IgcRingSeriesModule,
+    IgcItemLegendModule
 );
 
 export class DoughnutChartExplosion {
 
-    private chart: IgcDoughnutChartComponent;
-    private label: HTMLElement;
+    private chart: IgcDoughnutChartComponent; 
 
     constructor() {
 
-        let ringSeries = document.getElementById('ringSeries') as IgcRingSeriesComponent;
-        ringSeries.explodedSlices.add(3);
-        ringSeries.explodedSlices.add(4);
+        let ringSeries = document.getElementById('ringSeries') as IgcRingSeriesComponent;        
```

Comment:
why did you remove this code in DoughnutChart Explosion sample?
ringSeries.explodedSlices.add(3);	
ringSeries.explodedSlices.add(4);

Reply:
added back.

Reply:
it is still missing